src/components/Pages/OfficialDocuments/OfficialDocumentPage.js

```diff
@@ -0,0 +1,179 @@
+import React, { useState, useEffect  } from 'react'
+import { injectIntl } from 'react-intl'
+import { navigation as i18n2 } from 'js/i18n/definitions';
+import { isMobileQuery } from 'js/helpers/reactMediaQueries.js';
+
+import ChevronRight from 'components/SVGs/ChevronRight'
+import ChevronLeftBlue from 'components/SVGs/ChevronLeftBlue'
+import ChevronRightBlue from 'components/SVGs/ChevronRightBlue'
+import OfficialDocument from 'components/Pages/OfficialDocuments/OfficialDocument'
+
+const OfficialDocumentPage = ({ officialDocuments, intl }) => {
+
+  const documentsPerPage = 10
+  const isMobile = isMobileQuery()
+  const maxPagesShown = isMobile ? 4 : 7 // Desktop : Mobile (pagination pages shown)
+  const allDocs = officialDocuments.edges
+  const pages = buildPages()
+  const [ pageNumber, setPageNumber ] = useState(getHash())
+  const shownPages = buildPagination()
+  const page = pages[pageNumber]
+
+  useEffect(() => {
+    const str = typeof window !== 'undefined' ? window.location.hash.split("#")[1] : 0
+    const hash = (str > 0 && str <= pages.length) ? parseInt(str)-1 : 0
+    setPageNumber(hash)
+  })
+
+  useEffect(() => {
+    window.onpopstate = function(event) {
+      setPageNumber(getHash())
+      window.requestAnimationFrame( ()=> window.scroll(0,0) )
+    }
+    window.location.hash = pageNumber+1
+  })
+
+  function getHash(){
+    const str = typeof window !== 'undefined' ? window.location.hash.split("#")[1] : 0
+    const hash = (str > 0 && str <= pages.length) ? parseInt(str)-1 : 0
+    return hash
+  }
```

Comment:
All the use of the global `window` API is contained in these two methods. *useEffect* is bound to any changes to the localized `pageNumber` variable. 
- So, when the page change, the pager re-renders and that function fires. 

NOTE: I couldn't figure a reasonable way to embed the `typeof window !== 'undefined'` inside the *useEffect*...
-  Either I'd need an entirely new *useEffect* function, or it'd create an infinite loop   ∞